platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx

```diff
@@ -1,25 +1,66 @@
 ---
 title: Circle Configuration
-id: circle-configutration
+id: circle-configuration
 sidebar_position: 70
 ---
 
+## Using a Shared Circle Account Across All Tenants
+
 If the tenant was created with a [Circle] distribution account, then the tenant owner will need to manually configure that account from within the SDP dashboard.
 
 Once a user with owner privileges logs in, they will see a banner at the top of the page saying that the Circle account is pending configuration:
 
 ![Circle Configuration Banner](/assets/SDP/SDP30.png)
 
-Clicking on the banner will take the user to the Distribution Account section, where they can enter the Circle API key and the Circle Wallet ID.
+Clicking on the banner will take the user to the **Distribution Account** section, where they can enter the Circle **API Key** and the Circle **Wallet ID**.
 
 ![Circle Configuration](/assets/SDP/SDP31.png)
 
 :::info
 
-The API key will get stored in the database encrypted by the key `DISTRIBUTION_ACCOUNT_ENCRYPTION_PASSPHRASE`, while the Wallet ID is stored in plain text.
+The API key will be stored in the database encrypted by the key `DISTRIBUTION_ACCOUNT_ENCRYPTION_PASSPHRASE`, while the Wallet ID is stored in plain text.
 
 The Wallet ID is used to identify the Circle (internal) account when making disbursements. It's useful because a Circle account can have multiple wallets, each one with different currencies and balances.
 
 :::
 
+## Using Circle Sub-Accounts for Multi-Tenant Management
+
+In a multi-tenant environment, you may choose to use a single Circle account to manage funds for multiple agencies by utilizing Circle sub-accounts. This approach allows the main organization to maintain control over the primary funding source while providing each tenant with their own dedicated Circle wallet.
+
+### Steps to Configure Circle Sub-Accounts for Tenants
+
+#### 1. Create Circle Sub-Account
+
+Create a sub-account in Circle from your main wallet. This process is done within the Circle platform and is separate from SDP configuration. Please refer to <a href="https://developers.circle.com/circle-mint/create-a-sub-account" target="_blank" rel="noopener noreferrer">Circle's documentation</a> on how to create sub-accounts.
+
+#### 2. Create Tenant for the Agency
+
+In the SDP, add a new tenant for the agency. Refer to <a href="https://developers.stellar.org/platforms/stellar-disbursement-platform/api-reference/admin" target="_blank" rel="noopener noreferrer">the documentation on creating tenants</a> for guidance. As the initial owner, you will receive an invite link via email. Use this link to log in to SDP as the new tenant owner.
```

Comment:
`https://developers.stellar.org/platforms/stellar-disbursement-platform/api-reference/create-tenant` for create tenant call.